src/test/test_utils.hpp

```diff
@@ -79,6 +79,7 @@ const std::string passthrough_string_model_location = getGenericFullPathForSrcTe
 const std::string dummy_saved_model_location = getGenericFullPathForSrcTest(std::filesystem::current_path().u8string() + "/src/test/dummy_saved_model", false);
 const std::string dummy_tflite_location = getGenericFullPathForSrcTest(std::filesystem::current_path().u8string() + "/src/test/dummy_tflite", false);
 const std::string scalar_model_location = getGenericFullPathForSrcTest(std::filesystem::current_path().u8string() + "/src/test/scalar", false);
+const std::string no_name_output_model_location = std::filesystem::current_path().u8string() + "/src/test/no_name_output";
```

Comment:
This will likely not work on windows, see usage of `getGenericFullPathForSrcTest` above

Reply:
the tests on windows are passing

Reply:
That's strange. This new variable seems to have the same purpose as all the above and yet it works without wrapping in `getGenericFullPathForSrcTest` function? @rasapala can you take a look? Because to me, they all should be constructed in the same manner. Either with or without wrapper function.